package.json

```diff
@@ -38,11 +38,6 @@
     "refresh": "npm run clean && npm ci",
     "clean": "git clean -fX .eslintcache docs/build/ lib/ typings/"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm test && npm run lint"
```

Comment:
I'll change it to just `npm run build`, and if that turns out to be not enough, I'll merge this, how about that?

Reply:
Or perhaps `tsc --build docs/examples`, to avoid the extra `npm` process

Reply:
It's certainly going to improve it for other contributors, so that's a good idea.

I won't reinstall the hooks locally for me, though.

Reply:
`tsc --build docs/examples` is faster by almost .8s for me, so I'll go with that

Reply:
[Linting proved unreliable](https://github.com/telegraf/telegraf/pull/1239#issuecomment-743999803), and testing requires building first anyway.